### DISCUSSION TITLE
Qt and Commercial use

### DISCUSSION THREAD

**Discussion by z9484** [leon-thomm/Ryven#54] [Q&A] [answered]
Why does Ryven have a pop up saying Qt doesn't support Commercial Use? Pyside2 uses the LGPL which allows commercial use. I did a quick search and it looks like the only module imports Ryven uses are Qt Core, Qt Widgets and Qt Gui which are licensed under LGPL v3 as well as GPL, and commercial. The big GPL Qt module that I see people use a lot is QtCharts but I didn't find any references in Ryven.

https://www.qt.io/product/features#js-6-3 Qt's list of LGPL modules
https://www.e-education.psu.edu/geog489/node/2225 pyside2 vs pyqt

Great project by the way!

**Comment by leon-thomm** [comment, marked as answer]
you are totally correct, this is a relic, I don't know under what state of mind I was writing this, I mean the LGPL licensing is the reason for the existence of PySide. Anyway, I removed this misinformation in my dev version some time ago but didnt do that for the master branch, so I just did that, see [commit](https://github.com/leon-thomm/Ryven/commit/ecb6cf8cec29204ae66b30e0f55fbaa9e3401480). Sorry.
